packages/http-client/__tests__/proxy.test.ts

```diff
@@ -3,6 +3,7 @@
 import * as http from 'http'
 import * as httpm from '../lib/'
 import * as pm from '../lib/proxy'
+import {ProxyAgent} from 'undici'
```

Comment:
this line causes the "lightweight" client size to increase 3x times by 1Mb of code.